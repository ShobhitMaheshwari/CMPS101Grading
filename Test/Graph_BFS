https://github.com/ShobhitMaheshwari/CMPS101Grading/blob/master/Test/Graph_BFS.c
4 0
Check if graph is constructed and destructed properly
gcc -std=c99 -Wall -c Graph_BFS.c > w 2>&1
gcc -std=c99 -Wall -c main.c > w  2>&1
gcc -std=c99 -Wall -o Graph_BFS main.o Graph_BFS.o ../List.o ../Graph.o -pthread -lrt &> r
./Graph_BFS >  Graph_BFS.out
valgrind --leak-check=full --log-file="memoryLeak" ./Graph_BFS
