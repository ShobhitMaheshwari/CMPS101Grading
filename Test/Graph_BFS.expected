1
1
2
1
2
1
2
6
6
3
7
0
1
2
1
2
2
3
3
